gcloud-report/local-runner.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
This is well described in the tutorial, but it might be nice to add a comment about what it does/why for people stumbling upon this repo.

Reply:
I ended up stripping the local-runner stuff out of the tutorial, so I think an extensive comment on the file is useful. Will add!